src/modules/job-manager/jobtap.c

```diff
@@ -254,7 +254,7 @@ static int jobtap_remove (struct jobtap *jobtap,
     while (p) {
         const char *name = jobtap_plugin_name (p);
         if (all
-            || (isglob && fnmatch (arg, name, 0) == 0)
+            || (isglob && fnmatch (arg, name, FNM_PERIOD) == 0)
```

Comment:
Oh ha hah, FNM_PERIOD worked out nicely there.  Points for co-opting a file system convention.